app/models/name/merge.rb

```diff
@@ -29,23 +29,25 @@ def merge(old_name)
 
     # Move all misspellings over to the new name.
     old_name.misspellings.each do |name|
-      if name == self
-        name.correct_spelling = nil
-      else
-        name.correct_spelling = self
-      end
+      name.correct_spelling = if name == self
+                                nil
+                              else
+                                self
+                              end
```

Comment:
This seems like a legit use for the `? :` operator:
`name.correct_spelling = (name == self ? nil : self)`

Reply:
Quite right.